metricbeat/docs/fields.asciidoc

```diff
@@ -3193,7 +3193,7 @@ The hard limit on the number of file descriptors opened by the process. The hard
 [float]
 == cgroup Fields
 
-Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
+EXPERIMENTAL: Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
```

Comment:
@dedemorton If I use warning, a nice box is generated. Do you know if something similar exists for EXPERIMENTAL?